sites/partners/src/lib/helpers.ts

```diff
@@ -92,6 +91,12 @@ export const convertDataToPst = (dateObj: Date, type: ApplicationSubmissionType)
       time,
     }
   }
+
```

Comment:
(not blocking, just thinking out loud)

If the enum only has these values, I wonder if the linter will be smart enough to figure this out if we do a case statement instead of with these `ifs`

Reply:
Having a switch does solve this, thanks for the suggestion